ways_of_working.md

```diff
@@ -7,65 +7,63 @@ It further outlines expectations and responsibilities with regard to working on
 |---|---|---|---|---|
 | Kirstie Whitaker | PI | - | Oct 2018 - Present | - |
 | Malvika Sharan | Co-lead investigator | - | Jul 2021 - Present | previously: Community manager, Jan 2020 - Jun 2021 (100%) |
+| Emma Karoune | Community Manager | 5% | Jan 2021 - Present | - |
 | Arron Lacey | Community Manager | 10% | Nov 2021 - Present | - |
 | Achintya Rao | Community Manager | 10% | Nov 2021 - Present | - |
 | Vicky Hellon | Community Manager | 10% | Nov 2021 - Present | - |
 | Carlos Martinez-Ortiz | Netherlands eScience Center - RSE community manager | in-kind contribution | Feb 2020 - Present | - |
 | Esther Plomp | Faculty of Applied Sciences at Delft University of Technology - Data Steward | in-kind contribution | Feb 2020 - Present | - |
-| Sarah Gibson | 2i2c Research Software Engineer/JupyterHub Community Development | in-kind contribution | - | Prevously: Research Data Scientist, Nov 2018 - Aug 2021 (40%) |
-| May Yong | Research Associate | 40% | Oct 2021 - Dec 2021 | - |
-| Lydia France | Research Associate | 50% | Oct 2021 - Dec 2021 | - |
-| Iain Stanson | Research Associate | 30% | Nov 2021 - Dec 2021 | - |
+| Sarah Gibson | 2i2c Open Source Infrastructure Engineer/JupyterHub Community Development | in-kind contribution | - | Prevously: Research Data Scientist, Nov 2018 - Aug 2021 (40%) |
+| May Yong | Research Software Engineer | 40% | Oct 2021 - Dec 2021 | - |
+| Lydia France | Research Software Engineer | 50% | Oct 2021 - Dec 2021 | - |
+| Iain Stanson | Research Software Engineer | 30% | Nov 2021 - Dec 2021 | - |
 | Paul Owoicho | Google Season of Docs - Technical Writer | 100% | Sep 2020 - Dec 2020 | - |
```

Comment:
What do you think about formatting this table to be clearer about separating previous folks?

Not a strong opinion, if you think the dates are enough then that's totally fine with me.

Reply:
For this document, it makes sense to put them in separate tables so people can identify who to contact.